storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -708,59 +736,82 @@ static bool parse_args(int argc, char **argv) {
   return (0);
 }
 
-bool xbcloud_put(Object_store *store, const std::string &container,
-                 const std::string &backup_name, bool upload_md5) {
-  bool exists;
+typedef struct {
+  uint num;
+  std::atomic<bool> *has_errors;
+  struct file_list_t *file_list;
```

Comment:
would be nice if we can rename to global_file_list (because the file_list at different objects is confusing)

Reply:
Done